### PR TITLE
`Jobs` Fixes

### PR DESCRIPTION
**Pull request by patriciarealini**
## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._
seal penguin gif

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._
https://github.com/oddbird/oddleventy/pull/133
https://trello.com/c/lGqa2KUz/65-hire-javascript-developer

## Description
_The commit messages say what you did; this should explain why and/or how._
- [ ] I messed up and merged before getting jonny's review 🤦  see: https://github.com/oddbird/oddleventy/pull/133

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

## Show me
_Provide screenshots/animated gifs/videos if necessary._

### CODE REVIEW

**Review by mirisuzanne — APPROVED** [review]
LGTM. 

(Could match the existing line-break convention in quickstart?)

File: content/quickstart.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ we want the entire team involved in the conversation.
   of travel, lodging, and food when we get together —
   and pays a daily rate
   (currently \$400/day)
-  during the retreat.
+  during the retreat. (For the safety of our birds and our communities, due to the COVID pandemic our retreats are currently being conducted remotely.)
```
Comment (jgerigmeyer) [inline content/quickstart.md:139]:
```suggestion
  during the retreat.
  (For the safety of our Birds and our communities,
  due to the COVID pandemic
  our retreats are currently being conducted remotely.)
```

**Review by jgerigmeyer — APPROVED** [review]
🚀

File: content/quickstart.md (hunk 1 above)
Comment (jgerigmeyer) [reply content/quickstart.md]: Yeah, the linter doesn't (currently) touch linebreaks in `.md` files.

**Pull request merged**